CHANGELOG.md

```diff
@@ -129,6 +129,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - Fix issue with feature flags where default value may not be honored ([#12849](https://github.com/opensearch-project/OpenSearch/pull/12849))
 - Fix UOE While building Exists query for nested search_as_you_type field ([#12048](https://github.com/opensearch-project/OpenSearch/pull/12048))
+- Fix [BUG] Error invoking _stats with expand_wildcards on closed indices ([#13012](https://github.com/opensearch-project/OpenSearch/pull/13012))
```

Comment:
Please update the PR title and changelog entry to something like the following that would be end user understandable if reading patch notes:

```suggestion
- Improve error messages for `_stats` with closed indices ([#13012](https://github.com/opensearch-project/OpenSearch/pull/13012))
```